python/build-wheel-macos.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Cause the script to exit if a single command fails.
+set -e
+
+# Show explicitly which commands are currently running.
+set -x
+
+# Much of this is taken from https://github.com/matthew-brett/multibuild.
+# This script uses "sudo", so you may need to type in a password a couple times.
+
+MACPYTHON_URL=https://www.python.org/ftp/python
+MACPYTHON_PY_PREFIX=/Library/Frameworks/Python.framework/Versions
+GET_PIP_URL=https://bootstrap.pypa.io/get-pip.py
+DOWNLOAD_DIR=python_downloads
+
+PY_VERSIONS=("2.7.13"
+             "3.4.4"
+             "3.5.3"
+             "3.6.1")
+PY_INSTS=("python-2.7.13-macosx10.6.pkg"
```

Comment:
A possible alternative might be https://github.com/pyenv/pyenv. It lets you install 281 (!) different Python versions, but mostly through compilation from source, which is slow and might not do what you want.